packages/neo-one-smart-contract-compiler/src/test/getMonitor.ts

```diff
@@ -1,11 +0,0 @@
-import { DefaultMonitor, Monitor } from '@neo-one/monitor';
```

Comment:
Keep this one and remove the one in `__data__`. This is part of the external testing api, files in `__data__` should never be imported into anywhere except `__tests__`.